### PR TITLE
Add zbm-efi-kcl to release assets

### PR DESCRIPTION
**Pull request by zdykstra**
Tested as far as I could get locally. `zbm-efi-kcl` is successfully copied to the assets directory and signed.

### CODE REVIEW

**Comment by zdykstra** [comment]
Downloading allowed in https://github.com/zbm-dev/get.zfsbootmenu.org/commit/8cf6a7495e6a60e9ca7ba9efd30f5dbba4389b0e

I'll deploy that once v2.1.0 is actually released.

File: releng/tag-release.sh [hunk 1]
```diff
@@ -108,19 +108,28 @@ if echo "${release}" | grep -q "[A-Za-z]"; then
...
 # Create binary assets
 if ! releng/make-binary.sh "${release}"; then
   error "ERROR: unable to make release assets, exiting!"
 fi
 
+# Copy in any extra assets/files, relative to the project root
+# shellcheck disable=SC2043
+for extra in bin/zbm-efi-kcl ; do
+  [ -f "${extra}" ] || error "ERROR: missing ${extra}"
+  cp "${extra}" "${asset_dir}/"
```
Comment (ahesford) [inline releng/tag-release.sh:125]:
```suggestion
  cp "${extra}" "${asset_dir}/"
  assets+=( "${extra##*/}" )
```

File: releng/tag-release.sh [hunk 2]
```diff
@@ -108,19 +108,28 @@ if echo "${release}" | grep -q "[A-Za-z]"; then
   prerelease="--prerelease"
 fi
 
+[ -d "releng/assets/${release}" ] || mkdir -p "releng/assets/${release}"
+
+asset_dir="$( realpath -e "releng/assets/${release}" )"
+asset_files=()
```
Comment (ahesford) [inline releng/tag-release.sh:114]:
```suggestion
asset_files=()
assets=()
```

**Review by ahesford — CHANGES_REQUESTED** [review]
The extras ultimately need to make it into `asset_files` to be uploaded, but I haven't tested these changes any further than you have.

File: releng/tag-release.sh (hunk 1 above)
Comment (zdykstra) [reply releng/tag-release.sh:126]: Ahh yes. Missed that. I only checked that the signing script just globs the releng directory and signs everything in it.

**Pull request merged**